config/initializers/assets.rb

```diff
@@ -1,5 +1,14 @@
-Rails.application.config.assets.precompile += %w( bookreader.js )
-Rails.application.config.assets.precompile += %w( textdiff.js )
-Rails.application.config.assets.precompile += %w( openseadragon.js )
-Rails.application.config.assets.precompile += %w( datepicker/datepicker.js )
-Rails.application.config.assets.precompile += %w( intercom.js )
\ No newline at end of file
+# Be sure to restart your server when you modify this file.
+
+# Version of your assets, change this if you want to expire all your assets.
+Rails.application.config.assets.version = '1.0'
+
+# Add additional assets to the asset load path.
+# Rails.application.config.assets.paths << Emoji.images_path
+# Add Yarn node_modules folder to the asset load path.
+Rails.application.config.assets.paths << Rails.root.join('node_modules')
+
+# Precompile additional assets.
+# application.js, application.css, and all non-JS/CSS in the app/assets
+# folder are already added.
+# Rails.application.config.assets.precompile += %w( admin.js admin.css )
```

Comment:
This needs a joint review to make sure we are handling all of our js assets.  Can we load all of these as node_modules?  We had a hard time with the asset pipeline in our other (from scratch) rails 6 project.

I think this is going to take more work.